hive/runners/single_agent_loop.py

```diff
@@ -88,14 +117,14 @@ def run_one_step(self, observation, episode_metrics):
         if self._training:
             agent.update(copy.deepcopy(info))
 
-        self._transition_info.record_info(agent, info)
+        transition_info.record_info(agent, info)
         episode_metrics[agent.id]["reward"] += info["reward"]
         episode_metrics[agent.id]["episode_length"] += 1
         episode_metrics["full_episode_length"] += 1
 
         return terminated, truncated, next_observation
```

Comment:
Maybe I didn't understand the logic completely, but shouldn't we return the `transition_info` too?

Reply:
The transition_info object exists outside of this function and we are just changing the state of it, so I think this is fine, but if you can come up with a reason we should return the object, I'd be ok with changing it